components/automate-ui/src/app/pages/team-add-users/team-add-users.component.html

```diff
@@ -0,0 +1,50 @@
+<chef-page
+  subheading="Automate only displays local users."
```

Comment:
Hmm... probably more a question for @susanev: This is rather confusing. Shouldn't it say something more to the effect of "You can only add local users to a local team" ?

Reply:
good call, lets change it to.. `Only local users can be added to teams.`